test/new_relic/agent/transaction/trace_node_test.rb

```diff
@@ -108,7 +108,7 @@ def test_to_debug_str_one_child
     assert_equal(">>   0 ms [TraceNode] Custom/test/metric \n  >> 100 ms [TraceNode] Custom/test/other \n  << 150 ms Custom/test/other\n<< 200 ms Custom/test/metric\n", s.to_debug_str(0))
   end
 
-  def test_to_debug_str_multichild
+  def test_to_debug_str_multi_child
```

Comment:
This didn't seem to be used in enough places to justify treatment as a word (there were a number of things I accepted as merged...)